components/who-we-help.tsx

```
@@ -11,69 +12,76 @@ export function WhoWeHelp() {
           {cards.map((card, index) => (
-            <motion.div
-              key={index}
-              initial={{ opacity: 0, y: 20 }}
-              whileInView={{ opacity: 1, y: 0 }}
-              viewport={{ once: true }}
-              transition={{ duration: 0.6, delay: index * 0.2 }}
-              className="bg-card rounded-xl p-8 shadow-lg hover:shadow-xl transition-shadow duration-300"
+            <ShineBorder
+              className="relative size-48 rounded-lg"
+              color={["#A07CFE", "#FE8FB5", "#FFBE7B"]}
             >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)